curiouslearning/FeedTheMonsterJS
# fix: The monster animation does not go to idle state once the puzzle is completed (FM-345)

# Changes
- Fix the monster animation doest not go idle state once the puzzle is completed
- Merge monster to rive-monster-component
- Deleted the monster file
- Add test cases

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level
- Play the game
- Make sure the monster go to idle state every puzzle loaded
- Make sure monster will work properly in levelend and start scene

Ref: [FM-345](https://curiouslearning.atlassian.net/browse/FM-345)


[FM-345]: https://curiouslearning.atlassian.net/browse/FM-345?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
